portfolio/src/main/webapp/script.js

```diff
@@ -41,3 +41,36 @@ function toggleDescription(sectionId) {
   }
   toggleClass(sectionElement, "collapsed");
 }
+
+/**
+ * Change the slide to the current one + n
+*/
+function changeSlide(n, slider) {
+  if (!n || !Number.isInteger(n)) {
+    throw "changeSlide first argument is not an integer.";
+  }
+  let totalSlides = 0;
+  let currentlyDisplayed;
+  for (const elem of slider.children) {
+    const classes = elem.className.split(" ");
+    if (classes.indexOf("slide") !== -1) {
```

Comment:
Optional: how about extracting some utility functions
`hasClass(element: HtmlElement, className: string)`
and
`setClass(element: HtmlElement, className: string, value?: boolean)`?

Reply:
Actually even simpler:
`elemen.classList.add("d", "e", "f");` (also supports remove, has etc)
https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/add
(This only works in modern browsers but should be no problem in Chrome)

Reply:
I saw that function and I didn't use it for compatibility.

Reply:
I created the utility function and since the `elem.classList` is more clean, I used it.